onnxruntime/core/providers/js/operators/conv.h

```diff
@@ -9,14 +9,27 @@
 namespace onnxruntime {
 namespace js {
 
-template <bool is_channels_last>
+template <bool is_channels_last, bool is_fused_conv = false>
```

Comment:
nit: this approach of using a template for what could be a boolean arg to the ctor of the implementation is extremely inefficient in terms of binary size as the entire implementation is copied for each permutation of the template params. 

instead you can create lightweight wrapper classes to plugin the values.

e.g. change the base class ctor to `Conv(const OpKernelInfo& info, bool is_channels_last, bool is_fused_conv)` and add a templatized wrapper class to plugin the booleans. That way only the code of the wrapper class is templatized and there's a single copy of the implementation.

```c++
template <bool is_channels_last, bool is_fused_conv = false>
class ConvUsage : public Conv {
 public:
  ConvUsage(const OpKernelInfo& info) 
    : Conv(info, is_channels_last, is_fused_conv) {
};
```